examples/lora/lorawan/common/sx126x.go

```diff
@@ -23,23 +19,28 @@ var (
 	loraRadio *sx126x.Device
 )
 
+var (
+	spi                               = machine.SPI0
+	nssPin, busyPin, dio0Pin, dio1Pin = machine.GP17, machine.GP20, machine.GP11, machine.GP10
+	rxPin, txLowPin, txHighPin        = machine.GP13, machine.GP12, machine.GP12
+)
+
+func newRadioControl() sx126x.RadioController {
+	return sx126x.NewRadioControl(nssPin, busyPin, dio0Pin, dio1Pin, rxPin, txLowPin, txHighPin)
+}
+
 // do sx126x setup here
 func SetupLora() (lora.Radio, error) {
-	loraRadio = sx126x.New(machine.SPI3)
+	loraRadio = sx126x.New(spi)
```

Comment:
Why not moving SPI configuration in radio controller as well ?

Reply:
I just found that part a bit less obvious, also hoping to get some mock-based testing in place in a future iteration.